LICENSES_AND_LICENSE_HEADERS.md

```diff
@@ -0,0 +1,18 @@
+## LICENSES AND LICENSE HEADERS
```

Comment:
I think this file should be called `HEADERS.md` to keep it short.

Reply:
I like the idea of keeping it short but just trying to understand what other headers would we have other than licenses?

I didn't want to explicitly call out Elastic to be subtle as its the same for any opensource file which has an existing header this new header should be on top. Do you think its worth calling it out?

Reply:
I think it's worth calling out not to remove any original licenses and stuff like that, for example Elastic.

Reply:
Copyright headers are another thing. Not to confuse with license headers.

Reply:
Got it that makes more sense.